pkg/controller/bootstrap/bootstrapsigner.go

```diff
@@ -71,92 +71,103 @@ func DefaultBootstrapSignerOptions() BootstrapSignerOptions {
 
 // BootstrapSigner is a controller that signs a ConfigMap with a set of tokens.
 type BootstrapSigner struct {
-	client          clientset.Interface
-	configMapKey    string
```

Comment:
I would keep `configMapKey` and set it as part of `NewBootstrapSigner`. That way you can remove `getConfigMapKey` and just reference `configMapKey` directly.

You'll also need to keep `configMapName` and `configMapNamespace` as you've added them, so you can do the filtering.

Reply:
Sounds good, Done